src/test/kotlin/no/skatteetaten/aurora/mokey/service/HealthResponseParserTest.kt

```diff
@@ -0,0 +1,109 @@
+package no.skatteetaten.aurora.mokey.service
+
+import assertk.assertions.isEqualTo
+import com.fasterxml.jackson.databind.node.IntNode
+import com.fasterxml.jackson.databind.node.LongNode
+import com.fasterxml.jackson.databind.node.TextNode
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import no.skatteetaten.aurora.mokey.model.HealthPart
+import no.skatteetaten.aurora.mokey.model.HealthResponse
+import no.skatteetaten.aurora.mokey.model.HealthStatus
+import org.intellij.lang.annotations.Language
+import org.junit.jupiter.api.Test
+
+class HealthResponseParserTest {
+
+    @Test
+    fun `parse flat spring-boot 1-X health endpoint response`() {
+        @Language("JSON")
+        val json = """{
+  "status": "UP",
+  "atsServiceHelse": {
+    "status": "UP"
+  },
+  "diskSpace": {
+    "status": "UP",
+    "total": 10718543872,
+    "free": 10508611584,
+    "threshold": 10485760
+  },
+  "db": {
+    "status": "UP",
+    "database": "Oracle",
+    "hello": "Hello"
+  }
+}"""
+
+        val response = parse(json)
+        assertk.assert(response).isEqualTo(
+            HealthResponse(
+                HealthStatus.UP,
+                mutableMapOf(
```

Comment:
Her syns jeg det hadde vært ryddig å enten dele opp koden eller ha med named arguments. For min del begynner dette å bli såpass mye kode på en gang at jeg sliter litt med å lese det.

Reply:
Bare noen småting, du kan velge om du vil ta det med eller ikke :)